website/docs/topics/feature-flags/feature-flag-availability-over-consistency.md

```diff
@@ -0,0 +1,19 @@
+## 2. Design for failure. Favor availability over consistency.
```

Comment:
This might have to be quoted because it contains full stops. It might be alright, but it's safer with quotes. (The worst thing is that the build will fail with some inscrutable error message).

```suggestion
---
title: "2. Design for failure. Favor availability over consistency."
---
```